doc/nrf/known_issues.rst

```diff
@@ -979,6 +979,80 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-6-0
+
+DRGN-15817: Possible assert when used together with 802.15.4 protocols when running a scanner on nRF53 series
```

Comment:
I suggest to re-write:

DRGN-15817: Assertion during scanning in 802.15.4 multiprotocol configuration on nRF53 series

An assertion can may occur during scanning when the SoftDevice Controller is used in a 802.15.4 multiprotocol configuration.

@wiba-nordic ?